README.md

```diff
@@ -24,16 +24,16 @@ A frontend Framework for building single-page applications running in the browse
 
 * 👨‍👩‍👧‍👦 Powered by [Material UI](https://mui.com/material-ui/getting-started/), [react-hook-form](https://react-hook-form.com), [react-router](https://reacttraining.com/react-router/), [react-query](https://tanstack.com/query/latest/docs/framework/react/overview), [TypeScript](https://www.typescriptlang.org/) and a few more
 
-## Installation
+### Installation (Simplified)
```

Comment:
No, you're replacing the installation instructions for users by installation instructions for contributors. Instead, you should clarify that the installation instructions are for users, and leave the instructions unchanged.